content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md

```diff
@@ -68,9 +68,21 @@ Now that you have defined a stream, you can deploy it. To do so:
 
     ![Deployment Page](images/dataflow-deploy-http-ingest.png)
 
-    If you use the local Data Flow Server, add the following deployment property to set the port to avoid a port collision.
+<!--NOTE-->
 
-    ![Unique Port](images/dataflow-unique-port.png)
+If you use the local Data Flow Server, add the following deployment property to set the port to avoid a port collision.
+
+![Unique Port](images/dataflow-unique-port.png)
+
+<!--END_NOTE-->
+
+<!--NOTE-->
+
+If you use the Kubernetes Data Flow Server, add the deployment property `kubernetes.createLoadBalancer` with a value of `true` on the `http` source application to expose the service externally.
```

Comment:
There's only one server now anyway, so no need to present it as "Kubernetes Data Flow Server".